content/help/ops/networking/_index.md

```diff
@@ -4,3 +4,44 @@ description: Helps you manage the networking aspects of a running mesh.
 weight: 10
 type: section-index
 ---
+
+# Introduction 
+This section is intended as a guide to operators of an Istio based deployment.  It will provide information an Operator of a Istio deployment would need to manage the networking aspects of an Istio service Mesh.  Much of the information and many of the procedures that an Istio operator would require are already documented in other sections of the Istio documentation so this section will rely heavily on pointers to that other content.  Over time some of it may migrate here if necessary.
+## Key Istio Concepts
+When attempting to understand, monitor or troubleshoot the networking within an Istio deployment it is critical to understand the fundamental Istio concepts starting with the service mesh.  The service mesh is described in [Architecture](/docs/concepts/what-is-istio/#architecture).  As noted in the architecture section Istio has a distinct control plane and a data plane and operationally it will be important to be able to monitor the network state of both.  The service mesh is a fully interconnected set of proxies that are utilized in both the control and data plane to provide the Istio features.
+
+Another key concept to understand is how Istio performs traffic management.  This is described in [Traffic Management Explained](docs/concepts/#traffic-management).  Traffic management allows fine grained control with respect to what external traffic can enter or exit the mesh and how those requests are routed.   The traffic management configuration also dictates how requests between microservices within the mesh is handled.   Full details on how to configure the traffic management is available here: [Traffic Management Configuration](docs/tasks/traffic-management).
+
+TODO - Do we need to more completely describe gateways?  Not properly covered in concepts.
```

Comment:
We shouldn't be describing gateways here. If there's more gateway description needed, it needs to go in one of these sections:
* https://preliminary.istio.io/docs/concepts/traffic-management/#gateways
* https://preliminary.istio.io/docs/concepts/traffic-management/#ingress-and-egress
* https://preliminary.istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway

Gateway troubleshooting information [like his](https://github.com/istio/istio.github.io/blob/master/content/help/troubleshooting/index.md#route-rules-have-no-effect-on-ingress-gateway-requests) belongs in this doc.

Reply:
I agree,  the main concern was the gateways weren't properly described when I went looking for the details.  I think your above links are a start and I will stick to specific troubleshooting steps